spec/integration/manifest.spec.yml

```diff
@@ -363,6 +362,20 @@ spec:
       $ref: "#/definitions/categories"
     conditions:
       $ref: "#/definitions/conditions"
+    # requires a conditional JSON schema to update the value depending
+    # on the policy_templates length
+    policy_templates_behavior:
+      description: >
+        Behavior to manage the policy templates (policy_templates field) defined that must be available in Kibana as tiles.
+        This is just applicable for those packages that define more than one policy template. By default for these packages, Kibana
+        shows all the policy templates defined in the manifest separately plus another one referring to the
+        whole integration (that contains all the policy templates definitions) available to the user.
+
+        If the `integration_package_only` option is selected, then just one policy template for the whole integration containing all policy
+        templates will be available in Kibana. If the `policy_templates_only` option is selected, then just the policy templates defined
+        in the manifest will be available in Kibana without the specific policy template for the whole integration. If the `default` option
+        is selected, then Kibana keeps the same behavior described above.
+      type: string
```

Comment:
Not sure about the wording or even the naming for options here.
WDYT ? Any other suggestions?

Reply:
Proposed rewording in the comment above.